control0:
  ros__parameters:
    camera_node_name: "camera0"
    laser_node_name: "laser0"
    tracking_laser: [0.2, 0.0, 0.0]
    burn_color: [0.0, 0.0, 1.0]
    burn_time: 5
camera0:
  ros__parameters:
    video_dir: "/opt/video_stream"
    debug_video_dir: "/opt/debug_video_stream"
    save_video: False
    save_debug: False
    frame_period: .1
    rgb_size: [848, 480]
    depth_size: [848, 480]
laser0:
  ros__parameters:
    dac_type: "helios"
    dac_index: 0
    fps: 30
    pps: 30000
    transition_duration_ms: 0.5
